### PR TITLE
Us112465/dismiss pending

### PR DESCRIPTION
**Pull request by TomMichielsD2L**
image

Updated pending records, to be able to dismiss a pending record. Current implementation makes assumptions about the api.

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -135,6 +137,7 @@
   "to": "to",
   "total": "Total",
   "type": "Type",
+  "undismissRecord" : "Undismiss Record",
```
Comment (gravindran-d2l) [inline locales/en.json:140]: Maybe clear with Josh, but "undismiss" does not seem like a good word.

Comment (TomMichielsD2L) [reply locales/en.json:140]: I agree :D

File: src/components/cpd-pending-records.js [hunk 2]
```diff
@@ -110,12 +114,25 @@ class PendingRecords extends BaseMixin(LitElement) {
...
 	}
 
+	dismissRecordButtonClicked(e) {
+		const awardData = JSON.parse(e.target.getAttribute('data-award-id'));
+		if (this.filters.view === 'dismissed') {
+			this.cpdService.undismissRecord(awardData.IssuedAwardId);
+		}
+		else {
+			this.cpdService.dismissRecord(awardData.IssuedAwardId);
+		}
```
Comment (TomMichielsD2L) [inline src/components/cpd-pending-records.js:133]: Maybe we should re-fetch the pending at this point?

File: locales/en.json (hunk 1 above)
Comment (TomMichielsD2L) [reply locales/en.json]: Restore is the new word. It is a better word.

File: src/helpers/cpdRoutes.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export class CpdRoutes {
 	static CpdHome() { return '/d2l/custom/cpd/main'; }
 	static get CSVReport() { return 'record/report/csv'; }
+	static DismissRecord(awardId) { return `pending/dismiss${awardId}`; }
```
Comment (jwong-d2l) [inline src/helpers/cpdRoutes.js:4]:
```suggestion
	static DismissRecord(awardId) { return `pending/dismiss/${awardId}`; }
```

**Pull request merged**